src/js/util.dropchop.js

```diff
@@ -145,17 +159,28 @@ var dropchop = (function(dc) {
 
   dc.util.getBBox = function() {
     var bounds = dc.map.m.getBounds(),
-            sw = bounds.getSouthWest(),
-            ne = bounds.getNorthEast();
+        sw = bounds.getSouthWest(),
+        ne = bounds.getNorthEast();
     // we should probably check the size here?
     // node(57.7,11.9,57.8,12.0)
     return sw.lat+','+sw.lng+','+ne.lat+','+ne.lng;
   };
 
+  dc.util.getEsriBBox = function() {
+    var bounds = dc.map.m.getBounds(),
+        sw = bounds.getSouthWest(),
+        ne = bounds.getNorthEast();
+    return sw.lng+','+sw.lat+','+ne.lng+','+ne.lat;
+  };
+
   dc.util.uncollect = function(fc) {
     return fc.features[0];
   };
 
+  dc.util.esri2geo = function(arcjson) {
+    return toGeoJSON(arcjson);
+  };
```

Comment:
@npeihl Hey, I have a failing test:

```bash
FAILED TESTS:
  util.dropchop.js
    ✖ dc.util.esri2geo()
      PhantomJS 1.9.8 (Mac OS X 0.0.0)
    ReferenceError: Can't find variable: toGeoJSON
        at /Users/anthony/Projects/map-experiments/drop-n-chop/dist/static/js/dropchop.js:2
        at /Users/anthony/Projects/map-experiments/drop-n-chop/test/spec/util.spec.js:75
```

I think it comes down to this line, where does `toGeoJSON` come from?